paragon/tokens/global/color.json

```diff
@@ -0,0 +1,19 @@
+{
+  "color": {
+    "primary": {
```

Comment:
Defines overrides for the existing primary color levels from the Open edX theme.